tsconfig.base.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "@tsconfig/recommended/tsconfig.json",
+  "compilerOptions": {
+    "allowJs": true,
+    "checkJs": true,
+    "lib": ["ESNext"],
+    "module": "NodeNext",
+    "noEmit": true,
+    "strict": false,
+    "strictBindCallApply": true,
+    "strictFunctionTypes": true
```

Comment:
We want [`strictFunctionTypes`](https://www.typescriptlang.org/tsconfig#strictFunctionTypes) to confirm JSDoc `@param {string}` is not called with `number` etc